Cargo.toml

```diff
@@ -18,8 +18,9 @@ description = "Embeddable stream processing engine"
 [workspace.dependencies]
 denormalized = { path = "crates/core" }
 denormalized-common = { path = "crates/common" }
+denormalized-orchestrator = { path = "crates/orchestrator" }
 
-datafusion = { git = "https://github.com/probably-nothing-labs/arrow-datafusion", branch = "main" }
+datafusion = { git = "https://github.com/probably-nothing-labs/arrow-datafusion", branch = "amey/patch-with-node-id" }
```

Comment:
IMO we want to keep the node-id changes, we should merge them into https://github.com/probably-nothing-labs/arrow-datafusion main branch

Reply:
+1. makes sense to merge that one after the AggregateExprs PR.